### PR TITLE
Post tweet

### PR DESCRIPTION
**Pull request by jcorteza**

### CODE REVIEW

File: src/view/components/HeaderUI.jsx [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+    constructor(props) {
+        super(props);
+    }
+
+    render() {
+        return(
+            <header id="header">
+                <h1 id="headerH1Text">Lab for Josephine</h1>
+                <nav id="navbar">
+                        <NavLink exact to="/" activeStyle={{backgroundColor: "#1DA1F2", textDecoration: "underline"}}>Home Timeline</NavLink>
```
Comment (kev-chan) [inline src/view/components/HeaderUI.jsx]: is there way to do this without inline styles?

File: src/view/components/PostTweetUI.jsx [hunk 2]
```diff
@@ -0,0 +1,96 @@
...
+        this.handleChange = this.handleChange.bind(this);
+    }
+
+    handleChange(e) {
+        this.setState({ tweetText: e.target.value });
+    }
+
+    handleClick(e) {
+        e.preventDefault();
+        this.refs.postTweetButton.setAttribute("disabled", true);
```
Comment (kev-chan) [inline src/view/components/PostTweetUI.jsx]: does this need to be done here?

File: src/view/style/sass/main.scss [hunk 3]
```diff
@@ -3,31 +3,84 @@
...
+            text-align: center;
 
-        #headerHeaderText {
-            @include styled-font($weight: bold, $size: 40pt);
+            #headerH1Text {
+                @include styled-font($weight: bold, $size: 40pt);
+            }
+            
+            #navbar {
+                margin: 20px 0 20px;
```
Comment (kev-chan) [inline src/view/style/sass/main.scss]: you can shorthand this a bit more

File: src/view/style/sass/main.scss [hunk 4]
```diff
@@ -3,31 +3,84 @@
...
+                display: flex;
+                flex-direction: row;
+                justify-content: center;
+                min-height: fit-content;
+                background-color: #e8f5fd;
+                    
+                a {
```
Comment (kev-chan) [inline src/view/style/sass/main.scss]: don't use tags unless you absolutely need to

**Review by kev-chan — CHANGES_REQUESTED** [review]

**Comment by kev-chan** [comment]
- have coloring on the success/failure message
- don't allow resize of text area

File: src/view/components/PostTweetUI.jsx (hunk 2 above)
Comment (jcorteza) [reply src/view/components/PostTweetUI.jsx]: I'm not sure where else I could do this

**Pull request merged**